### DISCUSSION TITLE
How can I request access to become a contributor ?

### DISCUSSION THREAD

**Discussion by mibelbahri** [GoogleCloudPlatform/cloud-foundation-fabric#2116] [General]
How can I request access to become a contributor ? Thank you very much, 

I already have a small contrib in mind (already implemented)

**Comment by ludoo** [comment]
Hey, just send a PR. If you want to have a quick opinion before you invest time, open a discussion and we'll chime in. :) Remember to go through the CONTRIBUTING guide in the root of the repo!

**Comment by juliocc** [comment]
No need to request access. Simply send a PR and we'll review it.

We have a [contributing guide](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md) if you need help getting started.

Looking forward to your PR :)

**Comment by ludoo** [comment]
(thought this was an issue, let's keep this open since it's not) :)

**Reply by mibelbahri** [reply]
@ludoo @juliocc  thank you for your prompt answers, I can't seem to be able to push my branch ^^"

**Reply by mibelbahri** [reply]
In the contib doc it says:

> For small or first time issues the simplest way is to fork our repo, but if you are a regular contributor or are developing a large addition, ask us to be added directly to the repo so you can work on local branches, it makes life easier for both of us!

**Reply by ludoo** [reply]
you need to fork :) once / if you start contributing regularly, we can add you to the repo but the benefits are minimal
